src/Components/DayForm.js

```diff
@@ -61,9 +73,9 @@ export default function DayForm({ onSubmit, day }) {
               required
               maxlenght={20}
               type="text"
-              placeholder="add ur exercise"
-              //value=""
-              //onChange={}
+              placeholder="Dumbbell Incline Curl "
```

Comment:
This is just an example text right? Maybe write like this so its not confused as an already filled in input:
.. e.g. Dumbbell Incline Curl
but also there are more accessible ways to suggest inputs than placeholders. I posted an article about this in our channel a while ago :)

Reply:
Thx :)